javascript-if/main.js

```diff
@@ -0,0 +1,67 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH < 7) {
+    return 'acid';
+  } if (pH === 7) {
+    return 'nuetral';
+  } if (pH < 14) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
```

Comment:
If `ph` is `-20` this will return `acid` when it should return `invalid ph level`. Needs a little reworking on the logic.